extension/content.js

```diff
@@ -179,10 +179,12 @@ function addPatchDiffLinks() {
 }
 
 function removeDiffSigns() {
-	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
-		const textNode = element.childNodes[0];
-		textNode.nodeValue = textNode.nodeValue.replace(/^[+-]/, ' ');
+	const lines = $('.blob-code .blob-code-inner:not(.refined-github-diff-signs)');
+	lines.each((index, element) => {
+		const $element = $(element);
+		$element.html($element.html().replace(/^[+-]/, ''));
 	});
+	lines.addClass('refined-github-diff-signs');
```

Comment:
Any reason you chose to break this whole thing out and not leave it as an inline `each`?

Also, any reason you're not adding the class within the `each`?